components/navigation/navigation.css

```diff
@@ -0,0 +1,61 @@
+.nav-wrapper{
+    position: relative;
+    width: 90%;
+    border: 1px solid var(--color-black);
+}
+.navbar {
+    width: 100%;
+    background-color: var(--color-black);
+    padding: 1rem 1.5rem;
+    border-bottom: 0.3px solid var(--white);
+    position: sticky;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    /*set margin on divs below nav, nav eats content below it*/
+
+    display: flex;
+    flex-direction: row;
```

Comment:
display flex already have direction row. no need to write flex-direction row according to me

Reply:
Yes, It's redundant I'll make the change.